docs/docs/CPG/specs/dfg.md

```diff
@@ -54,6 +54,49 @@ Scheme:
     expression -- DFG --> node;
 ```
 
+
+## AssignExpression (`operatorCode: =`)
```

Comment:
@oxisto I guess we also have to add a documentation for `+=`, etc., right? And we should probably delete the part in the `BinaryOperator` even if I believe we do not prevent a user from modeling an assignment that way (I think we can't block `=` as an operator in case some language uses this in another way)

Reply:
We actually *do* block `=` and all other compound operators in the `BinaryOperator` by throwing a translation exception. So yes, it should be removed from `BinaryOperator` and compound assignments should be added to `AssignExpression`.

Reply:
ok...I tried to adapt the specs. Maybe, throwing the translation exception might not be a good idea for each language... 

Do we support `+=` with multiple elements on `lhs` and `rhs`?

Reply:
No, compound assignments are only allowed with one lhs and rhs